apps/teams-test-app/src/App.tsx

```diff
@@ -17,7 +17,7 @@ const getOriginsParam = urlParams.has('origins') && urlParams.get('origins') ? u
 const validMessageOrigins: string[] | undefined = getOriginsParam ? getOriginsParam.split(',') : undefined;
 
 // This is added for custom initialization when app can be initialized based upon a trigger/click.
-if (!urlParams.has('customInit') || !urlParams.get('customInit')) {
+if (!urlParams.has('customInit') || !urlParams.get('customInit') || !urlParams.get('precacheApp')) {
```

Comment:
do we need to check `urlParams.has` before calling `urlParams.get`?

Reply:
Nvmd yes we need. I thought it was another line. Thank you for pointing it out.

Reply:
Updated!